### PR TITLE
Clarify roles

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

Clarifying based on discussion in https://github.com/Azure-Samples/azure-search-openai-demo/issues/4

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by charris-msft** [comment]
The solution is ugly (not your fault), but the update looks good to me.

**Comment by pamelafox** [comment]
@charris-msft Please approve formally (this repo requires approvals before merging)

**Review by charris-msft — APPROVED** [review]
LGTM

**Pull request merged**
